gettsim/transfers/arbeitsl_geld_2/arbeitsl_geld_2.py

```diff
@@ -37,7 +38,7 @@ def arbeitsl_geld_2_m_hh(
         wohngeld_vorrang_hh
         | kinderzuschlag_vorrang_hh
         | wohngeld_kinderzuschlag_vorrang_hh
-        | rentner_in_hh
+        | (anz_erwachsene_hh == anz_rentner_hh)
```

Comment:
Currently, Grundsicherung im Alter is only payed if all adults in hh are retired and ALG2 otherwise.

See also the [discussion](https://gettsim.zulipchat.com/#narrow/stream/243798-arbeitsl_geld_2-etc.2E/topic/ALG.202.20if.20partner.20is.20retired/near/260008404) I opened on Zulip.